linters/.eslintrc

```diff
@@ -1,5 +1,16 @@
 // Use this file as a starting point for your project's .eslintrc.
 // Copy this file, and add rule overrides as needed.
 {
-  "extends": "airbnb"
+  "extends": ["airbnb", "plugin:jest/recommended"],
+  "rules": {
+    "semi": ["error", "never"],
+    "max-len": [1, 80, 2, {
+      "ignorePattern": "^import\\s.+\\sfrom\\s.+;$",
+      "ignoreUrls": true
+    }],
+    "no-multiple-empty-lines": [2, { "max": 1, "maxEOF": 1 }]
+  },
+  "env": {
+   "jest": true
```

Comment:
This is far too prescriptive to ever go in this repo; users can use whatever test runner they like. Additionally, you’d never want this env enabled for your non-test files.

Reply:
What is the best way for me to get this done?

Reply:
You can (and should, if you're using jest) add the `jest` env to your own eslint config, using "overrides" so that it only targets your test files.